modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,534 @@
+
+[[redpanda-reqs]]
+=== Check if the system meets Redpanda requirements
+
+Run a system check to ensure that your system meets Redpanda's requirements.
+
+Example:
+
+```bash
+sudo rpk redpanda check
+```
+
+Output:
+```bash
+System check results
+CONDITION                                          REQUIRED      CURRENT          SEVERITY  PASSED
+Ballast file present                               true          true             Warning   true
+Clock Source                                       tsc           tsc              Warning   true
+Config file valid                                  true          true             Fatal     true
+Connections listen backlog size                    >= 4096       4096             Warning   true
+Data directory filesystem type                     xfs           xfs              Warning   true
+Data directory is writable                         true          true             Fatal     true
+Data partition free space [GB]                     >= 10         1755.29          Warning   true
+Dir '/var/lib/redpanda/data' IRQs affinity set     true          true             Warning   true
+Dir '/var/lib/redpanda/data' IRQs affinity static  true          true             Warning   true
+Dir '/var/lib/redpanda/data' nomerges tuned        true          true             Warning   true
+Dir '/var/lib/redpanda/data' scheduler tuned       true          true             Warning   true
+Free memory per CPU [MB]                           2048 per CPU  7659             Warning   true
+Fstrim systemd service and timer active            true          true             Warning   true
+I/O config file present                            true          true             Warning   true
+Kernel Version                                     3.19          5.15.0-1056-aws  Warning   true
+Max AIO Events                                     >= 1048576    1048576          Warning   true
+Max syn backlog size                               >= 4096       4096             Warning   true
+NIC IRQs affinity static                           true          true             Warning   true
+NTP Synced                                         true          true             Warning   true
+RFS Table entries                                  >= 32768      32768            Warning   true
+Swap enabled                                       true          true             Warning   true
+Swappiness                                         1             1                Warning   true
+Transparent huge pages active                      true          true             Warning   true
+```
+
+[[redpanda-lmr]]
+=== Redpanda's latest version
+
+Check that Redpanda is running on the xref:https://github.com/redpanda-data/redpanda/releases[latest maintenance release^] on every node.
+
+Example:
+```bash
+/usr/bin/redpanda --version
+```
+
+Output:
+```bash
+v23.3.9 - f0a24fa4b60d55bd075c291e213807b5dfa74c0a
```

Comment:
any way to get this to track the version detail of the major version number we're on? I don't know if we have an adoc tag that can dereference at build time to be "latest" or whatever.

Reply:
Updated it to 24.1.2. What's the other value? Commit sha?

Reply:
I believe its version - commit sha. We have something in docs to use for this